website-guidelines.md

```diff
@@ -30,12 +30,16 @@ this vetting to CNCF staff if it becomes a burden.
 4. Links to companies offering support are expected to go a page that at least mentions support of
 the project. This can either be the company homepage or a project-specific landing page.
 5. If there is a copyright notice at the bottom of the page, copyright should be to the project authors
-or to CNCF, not the origin company.
+or to CNCF, not the origin company. For details, see [Copyright notices](copyright-notices.md).
 6. CNCF requests that graduated and incubating projects include the sentence “We are a Cloud Native
 Computing Foundation project.” and the CNCF logo near the bottom of their project homepage.
 Sandbox-level projects should include the sentence “We are a Cloud Native Computing Foundation
 sandbox project.” and the CNCF logo. We also appreciate a link to KubeCon + CloudNativeCon as the
 events approach.
-7. All projects must contain a website footer message referencing trademark guidelines, "Copyright The Linux Foundation. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our [Trademark Usage](https://www.linuxfoundation.org/trademark-usage/) page."
```

Comment:
I removed "Copyright The Linux Foundation" because it was inconsistent with the guidelines given in [Copyright notices](https://github.com/cncf/foundation/blob/master/copyright-notices.md).